microlith/grafana/provisioning/dashboards/bucket-overview.json

```diff
@@ -45,7 +44,10 @@
   "liveNow": false,
   "panels": [
     {
-      "datasource": "-- Mixed --",
+      "datasource": {
+        "type": "datasource",
+        "uid": "-- Mixed --"
+      },
```

Comment:
@bentonam what version of Grafana were you running when you made these? Because this looks like [a change from Grafana 8.3](https://github.com/grafana/grafana/pull/33817), while CMOS is still on 8.2 (because we'd need to make this exact change when upgrading).

Reply:
#228 will upgrade us to 8.3.